### DISCUSSION TITLE
For OneraM6, objective function is not decreasing at all!!

### DISCUSSION THREAD

**Discussion by Mohsin-Bashir** [mdolab/dafoam#632] [Tutorials]
I ran oneraM6 tutorial without any changes but according to opt_IPOPT.txt, objective function does not seem to change at all.
[logOpt.txt](https://github.com/mdolab/dafoam/files/15036703/logOpt.txt)
[opt_IPOPT.txt](https://github.com/mdolab/dafoam/files/15036705/opt_IPOPT.txt)

**Comment by friedenhe** [comment]
Your adjoint did not converge. I am not sure why this was the case cause it converges well on my end..
